sympy/solvers/ode/tests/test_single.py

```diff
@@ -140,22 +140,26 @@
 """
 
 
+def _get_example(solver, example):
+    return {
+            'eq': solver['examples'][example]['eq'],
+            'sol': solver['examples'][example]['sol'],
+            'XFAIL': solver['examples'][example].get('XFAIL', []),
+            'func': solver['examples'][example].get('func',solver['func']),
+            'example_name': example,
+            'slow': solver['examples'][example].get('slow', False),
+            'simplify_flag':solver['examples'][example].get('simplify_flag',True),
+            'checkodesol_XFAIL': solver['examples'][example].get('checkodesol_XFAIL', False),
+            'dsolve_too_slow':solver['examples'][example].get('dsolve_too_slow',False),
+            'checkodesol_too_slow':solver['examples'][example].get('checkodesol_too_slow',False),
+        }
+
```

Comment:
@oscarbenjamin were you suggesting like this?

Reply:
The general idea here is that individual examples are represented by a dict. There can just be a function that takes the dict for an individual example and fills in the missing keys. Then we can build that into a decorator.

I think it would make most sense to do this at the moment the examples are created with a decorator like:
```
@_add_example_keys
def _get_examples_separable():
    ...
```

Reply:
@oscarbenjamin does this look correct way?